File: procedural-programming/homework-001/c/exercise-035.c
/* Student: Pedro Henrique Resende Ribeiro
 * Description: Calculating the hypotenuse of a triangle
 * Date: 2024-01-09 - Time: 01:20
 */

#include <math.h>
#include <stdio.h>

int main() {

    float a, b, hypotenuse;

    printf("Enter the 1st side of the triangle: ");
    scanf("%f", &a);

    printf("Enter the 2nd side of the triangle: ");
    scanf("%f", &b);

    hypotenuse = sqrt(pow(a, 2.0) + pow(b, 2.0));

    printf("The hypotenuse is %f\n", hypotenuse);

    return 0;

}

File: procedural-programming/homework-001/python/exercise_035.py
# Student: Pedro Henrique Resende Ribeiro
# Description: Calculating the hypotenuse of a triangle
# Date: 2024-01-09 - Time: 01:20

from math import sqrt

a = float(input('Enter the 1st side of the triangle: '))
b = float(input('Enter the 2nd side of the triangle: '))

hypotenuse = sqrt(a ** 2.0 + b ** 2.0)

print(f'The hypotenuse is {hypotenuse}')
